js/home.js

```diff
@@ -6,6 +6,13 @@ window.onload = function() {
   // display info message if the user is coming for the first time
   if (wkUserData.userPublicKey === undefined || wkUserData.userPublicKey == ""){
     document.querySelector(".info").style.display = 'inline';
+    if (window.location.href.includes('wanikani.com/settings/account')) {
+      alert("Please enter your API Key.")
+    } else {
+      alert("Redirecting to WaniKani...")
+      window.open('https://www.wanikani.com/settings/account/#public-api-key',
```

Comment:
I thought about intrusiveness.. In fact, my idea at first was to have it automatically fetch the API key! :rofl: I thought that was too intrusive. Now that I think about an automatic redirect, that seems intrusive also. 

I just got a free lesson on UX design!